### PR TITLE
Add Sound Blaster Control App

### PR DESCRIPTION
**Pull request by iMonZ**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Failing with

> uninstall script /Applications/Creative/Uninstaller.app/Contents/MacOS/Uninstaller does not exist.

**Comment by ran-dall** [comment]
@iMonZ Where you going to take a look at this, or should we proceed with closing this out?

**Comment by iMonZ** [comment]
Oh I’m sorry I thought this was closed!
Im fixing this today

**Comment by vitorgalvao** [comment]
Needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers), it seems. We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

What are the names of the apps this installs under `/Applications/Creative`?

**Comment by iMonZ** [comment]
> Needs to be submitted to homebrew/cask-drivers, it seems. We define drivers as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

It‘s the same kind as #83746 and this is also an Cask (because this is an control App for this device)


- [ ] 


> What are the names of the apps this installs under /Applications/Creative?

This will be automatically uninstalled with the installer script!

**Comment by vitorgalvao** [comment]
> It‘s the same kind as #83746

Which looks like it too should be in the drivers repo.

**Comment by iMonZ** [comment]
> /Applications/Creative
Creative Uninstaller.app
Product Registration.app
Sound Blaster Control Panel.app

**Comment by vitorgalvao** [comment]
> It‘s the same kind as #83746

Which looks like it too should be in the drivers repo.

> because this is an control App for this device

If it controls *a device*, it definitely belongs in the other repo.

> This will be automatically uninstalled with the installer script!

Not why I’m asking. I really do want the names of the apps.

File: Casks/sound-blaster-control.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'sound-blaster-control' do
```
Comment (vitorgalvao) [inline Casks/sound-blaster-control.rb:1]: Would be more compliant with the rules to be `sound-blaster-control-panel`.

**Comment by reitermarkus** [comment]
@iMonZ, please rebase this and adjust the style to the new rules.

**Comment by vitorgalvao** [comment]
Definitely looks like it belongs in [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). I’ll move the other cask.

There’s been no reply on this for over a month, so I’ll close it. It can be resubmitted to the correct repo with the requested fixes.

**Comment by vitorgalvao** [comment]
> @iMonZ, please rebase this and adjust the style to the new rules.

User rebases make it harder to see what changed between commits, which means we have to reevaluate the whole cask every time. Since cask submissions should always be a single commit, it’s better if we do it ourselves through the Github interface or [a script](https://github.com/vitorgalvao/tiny-scripts/blob/master/climergebutton) (that’s an example).

**Pull request closed without merging**